### ISSUE TITLE
Add ccmake to python wheel

### ISSUE THREAD

**Issue by SteffenSeckler** [scikit-build/cmake-python-distributions#66]
### Feature request
I would like to see the curses gui (ccmake) integrated into the python wheel. 
I am guessing that only the necessary requirements are missing for that.

**Comment by jcfr** [comment]
Good idea. If you can submit a PR, it will be reviewed and integrated next
week when back in the office.

**Comment by jcfr** [comment]
There is now a PR adding support for distributing and executing ccmake if available.

> I am guessing that only the necessary requirements are missing for that.

Indeed, remaining work would be to enable the `BUILD_CursesDialog` option.
See https://github.com/scikit-build/cmake-python-distributions/blob/aa9a0fd10838be4de6112a953924a183fec44c2a/CMakeLists.txt#L186-L205

Since I am not sure all prerequisites are available in the manylinux dockcross images used to create the linux wheel, some more work may be needed to have the curse devel libraries available in the build environment.

**Comment by HumHongeKamyaab** [comment]
Any update on this issue?
